lib/simple-auth.js

```diff
@@ -14,20 +14,23 @@ module.exports = simpleAuth;
 // Start session. Only anonymous handshakes are allowed.
 //   connection - JSTP connection
 //   application - application instance
-//   username - null (required by the auth callback contract)
-//   password - null (required by the auth callback contract)
+//   strategy - authentication strategy (only 'anonymous' is supported)
+//   credentials - authentication credentials
 //   callback - callback function that has signature (error, sessionId)
```

Comment:
Callback signature here does not correspond to the callback you are calling on line 33.